scripts/list_outdated_testcases.py

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env python
+
+# -*- coding: utf-8 -*-
+# pylint: disable=logging-format-interpolation
+"""
+Compare output of 'pytest --collect-only' with data in Polarion.
+
+The outcome is a CSV file with test cases that might need to be
+deactivated/deleted in Polarion.
+
+Usage:
+    Export a CSV file with _automated_ test cases from Polarion and use it as
+    an input for this script.
+
+IMPORTANT: double-check that the test case is really missing in pytest and was
+not just uncollected because of provider type, appliance version, etc.
+"""
+
+from __future__ import unicode_literals, absolute_import
+
+import argparse
+import codecs
+import csv
+import logging
+import operator
+import os
+import re
+import sys
+
+from cStringIO import StringIO
+from contextlib import contextmanager
+
+from dump2polarion.csvtools import get_imported_data
+
+
+# pylint: disable=invalid-name
+logger = logging.getLogger(__name__)
+
+
+class CompareException(Exception):
+    """testcases exception."""
```

Comment:
I wouldn't say this is recommended. But I agree I missed that point that docstring could be enough in such case.